integration/additional_bindings/Makefile

```diff
@@ -11,7 +7,7 @@ clean:
 	find . -regex "\.\/.*\/.*\.go" -exec rm {} +
 	rm -f main
 
-protoc:
+protoc: protoc-build
```

Comment:
I don't like this `protoc-build` in every sub-test - maybe it's better to run them once in the core Makefile.

And if you want to run a single test individually - just prepare your environment manually running `make protoc-build` and then run `make test`.

Reply:
or we could do even better and make 2 targets.
`test` would explicitly run `protoc-build` - you would use it in manuall run
`test-ci` won't call `protoc-build` - it would be used in the core Makefile to run all tests.

Reply:
discussed verbally